plugins/com.google.cloud.tools.eclipse.appengine.standard.java8.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,14 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Google Cloud Tools for App Engine Standard Java 8 Tests
+Bundle-SymbolicName: com.google.cloud.tools.eclipse.appengine.standard.java8.test
+Fragment-Host: com.google.cloud.tools.eclipse.appengine.standard.java8
+Bundle-Version: 0.0.1.qualifier
+Bundle-Vendor: Google, Inc.
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Import-Package: com.google.cloud.tools.appengine;version="0.2.11",
```

Comment:
The package versions are still at 0.2.11!  I wonder if the maven-bundle-plugin compares signature changes if the previous jar is found?